package.json

```diff
@@ -12,7 +12,7 @@
     "@tsconfig/node16": "^16.1.3",
     "@types/jest": "^29.5.12",
     "@types/node": "^20.12.7",
-    "eslint": "8.57.0",
+    "eslint": "9.2.0",
```

Comment:
`typescript-eslint` still requires `eslint@8.57.0`, it can work with 9.x but yarn throws a warning.

Reply:
I think we need to close this MR, it thinks that it's going from 9 -> 9.2 when it's really doing 8.x to 9.x, and the next check will probably fail since it bumps a major version.